router/handle_lifecycle.go

```diff
@@ -127,8 +127,10 @@ func (rt *Handle) Setup(
 	rt.routerTransformInputCountStat = stats.Default.NewTaggedStat("router_transform_num_input_jobs", stats.CountType, statTags)
 	rt.routerTransformOutputCountStat = stats.Default.NewTaggedStat("router_transform_num_output_jobs", stats.CountType, statTags)
 	rt.batchInputOutputDiffCountStat = stats.Default.NewTaggedStat("router_batch_input_output_diff_jobs", stats.CountType, statTags)
-	rt.processJobsRequestsHistogramStat = stats.Default.NewTaggedStat("router_process_jobs_requests", stats.HistogramType, statTags)
-	rt.processJobsRequestsCountStat = stats.Default.NewTaggedStat("router_process_jobs_count", stats.CountType, statTags)
+	rt.processJobsHistogramStat = stats.Default.NewTaggedStat("router_process_jobs", stats.HistogramType, statTags)
+	rt.processJobsCountStat = stats.Default.NewTaggedStat("router_process_jobs_count", stats.CountType, statTags)
+	rt.processRequestsHistogramStat = stats.Default.NewTaggedStat("router_process_requests", stats.HistogramType, statTags)
+	rt.processRequestsCountStat = stats.Default.NewTaggedStat("router_process_requests_count", stats.CountType, statTags)
```

Comment:
If you're creating new histograms please update the [buckets file](https://github.com/rudderlabs/rudder-server/blob/379fcbd9f9d22366dc067088608069df9685dd2e/runner/buckets.go#L1-L0).